### PR TITLE
Update API certification serializers

### PR DESCRIPTION
**Pull request by romanblanco**
updating to reflect the changes in https://github.com/pulp/pulp_ansible/pull/237

### CODE REVIEW

File: galaxy_api/api/ui/serializers/collection.py [hunk 1]
```diff
@@ -82,7 +82,8 @@ class _CollectionSerializer(serializers.Serializer):
     name = serializers.CharField()
     download_count = serializers.IntegerField(default=0)
     latest_version = CollectionVersionSerializer(source='*')
-    is_certified = serializers.BooleanField()
+    certification = serializers.ChoiceField(
```
Comment (cutwater) [inline galaxy_api/api/ui/serializers/collection.py]: Minor: Normally I would prefer enum for list of acceptable string values.

Otherwise LGTM.

Comment (romanblanco) [reply galaxy_api/api/ui/serializers/collection.py]: The documentation to [ChoiceField](https://www.django-rest-framework.org/api-guide/fields/#choicefield) referres to the parameter as a _list_, but surely I can try if _enum_ works too.

Comment (romanblanco) [reply galaxy_api/api/ui/serializers/collection.py]: @cutwater I've applied the suggestions to use enum and updated the PR. Please try again the changes if they work for you.

Comment (romanblanco) [reply galaxy_api/api/ui/serializers/collection.py]: @cutwater Taking back using enum

**Pull request merged**